pyproject.toml

```diff
@@ -17,6 +17,8 @@ pystac = "^1.2.0"
 jsonschema = "^4.2.1"
 jsonschema-rs = "^0.13.1"
 Shapely = "^1.8.0"
+requests = "^2.26.0"
```

Comment:
How come we need these two libs?

Reply:
It crashed an error saying that they are required for fsspec

Reply:
ahh ok! so they are optional deps for fsspec